vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/test/grid-connector-sorting.test.ts

```diff
@@ -53,7 +53,7 @@ describe('grid connector - sorting', () => {
 
       sorters[0].click();
       expect(grid.$server.sortersChanged).to.be.calledOnce;
-      expect(grid.$server.sortersChanged.args[0][0]).to.eql([{ path: 'name', direction: null }]);
+      expect(grid.$server.sortersChanged.args[0][0]).to.eql([]);
```

Comment:
**note:** The new implementation of the web component removes sorters without direction from the request in single column sorting mode:

https://github.com/vaadin/web-components/commit/37fdac81fa4e6630769474f118391c52708315c4#diff-3c17927c07b9009af8f21234efdca93c79dfb5095cb182cff0381a875074ebcaR191-R193